lib/monaco-adapter.js

```diff
@@ -202,6 +202,17 @@ var MonacoAdapter = function () {
       return cursor.clientID === clientID;
     });
 
+    if (otherCursor) {
+      // Recreate the cursor if the color or the name has changed
+      if (otherCursor.name !== name || otherCursor.color !== color) {
+        this.removeOtherCursorDecoration(otherCursor)
+        this.otherCursors.splice(this.otherCursors.indexOf(otherCursor), 1)
+        otherCursor = null
+      } else {
+        otherCursor.position = cursor.position
+        otherCursor.selectionEnd = cursor.selectionEnd
```

Comment:
I'm not sure why you moved these 2 lines here

Reply:
because we already check if `otherCursor` exists so I wanted to have all the logic to handle existing cursor at the same place. But I'd be ok to leave it in the else below too

Reply:
it's fine!